close_test.go

```diff
@@ -7,19 +7,19 @@ import (
 	"testing"
 	"time"
 
-	"github.com/ONSdigital/log.go/log"
+	"github.com/ONSdigital/log.go/v2/log"
 	. "github.com/smartystreets/goconvey/convey"
 )
 
 func TestSuccessfulCloseMongoSession(t *testing.T) {
 	_, err := setupSession()
 	if err != nil {
-		log.Event(nil, "mongo instance not available, skip close tests", log.ERROR, log.Error(err))
+		log.Error(context.Background(), "mongo instance not available, skip close tests", err)
```

Comment:
I know this isn't within the scope of this PR, but why are test just being skipped when things are failing? How can you be certain that every time it fails here you wanted it to fail?

Reply:
good question, and I don't know. @DavidSubiros are you able to answer this?

Reply:
I think there were some tests doing real connections, I would suggest mocking the real connection so that the unit tests don't depend on external things. 
I imagine that here, if the real TCP connection could not be established, then we don't attempt to Close it. I think we should create some mocks (or, if it's simpler/clearer, create a function that establishes the connection in a real scenario, and is replaced with a stub in the test).

Reply:
Is this something that needs to be captured in a ticket? (i'm not sure who maintains this library, apologies)